util/execdetails/execdetails.go

```diff
@@ -206,6 +211,24 @@ func (sd *ScanDetail) Merge(scanDetail *ScanDetail) {
 	sd.WaitTime += scanDetail.WaitTime
 }
 
+func (sd *ScanDetail) String() string {
+	buf := bytes.NewBuffer(make([]byte, 0, 16))
+	if sd != nil {
+		buf.WriteString(ProcessTimeStr + ":" + strconv.FormatFloat(sd.ProcessTime.Seconds(), 'f', -1, 64))
+		buf.WriteString(", " + WaitTimeStr + ":" + strconv.FormatFloat(sd.ProcessTime.Seconds(), 'f', -1, 64))
+		buf.WriteString(", rocksdb{")
+		buf.WriteString(ProcessKeysStr + ": " + strconv.FormatInt(sd.ProcessedKeys, 10))
+		buf.WriteString(", " + TotalKeysStr + ": " + strconv.FormatInt(sd.TotalKeys, 10))
```

Comment:
These two should be put outside rocksdb

Reply:
They have been put outside rocksdb.

Reply:
> They have been put outside rocksdb.

Did you forget to push the code? According to the PR I'm reviewing now, the constructed string looks like `....., rocksdb{Process_keys: ..., Process_keys: ..., ....}`.

Reply:
Oh, sorry. I will put it outside.

Reply:
How about display like: 

```sql
scan_detail: {total_process_keys: 10, total_keys: 15, rocksdb: {delete_skipped_count: 5, key_skipped_count: 1, block: {cache_hit_count: 10, read_count: 20, read_byte: 1.50 MB}}}
```

@breeswish @TszKitLo40 PTAL

```go
func (sd *ScanDetail) String() string {
	buf := bytes.NewBuffer(make([]byte, 0, 16))
	buf.WriteString("scan_detail: {")
	buf.WriteString("total_process_keys: ")
	buf.WriteString(strconv.FormatInt(sd.ProcessedKeys, 10))
	buf.WriteString(", total_keys: ")
	buf.WriteString(strconv.FormatInt(sd.TotalKeys, 10))
	buf.WriteString(", rocksdb: {")
	buf.WriteString("delete_skipped_count: ")
	buf.WriteString(strconv.FormatUint(sd.RocksdbDeleteSkippedCount, 10))
	buf.WriteString(", key_skipped_count: ")
	buf.WriteString(strconv.FormatUint(sd.RocksdbKeySkippedCount, 10))
	buf.WriteString(", block: {")
	buf.WriteString("cache_hit_count: ")
	buf.WriteString(strconv.FormatUint(sd.RocksdbBlockCacheHitCount, 10))
	buf.WriteString(", read_count: ")
	buf.WriteString(strconv.FormatUint(sd.RocksdbBlockReadCount, 10))
	buf.WriteString(", read_byte: ")
	buf.WriteString(memory.FormatBytes(int64(sd.RocksdbBlockReadByte)))
	buf.WriteString("}}}")
	return buf.String()
}
```

Reply:
LGTM